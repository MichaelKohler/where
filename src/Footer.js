import React from 'react';
import './scss/footer.scss';

const Footer = () => {
  return (
    <footer>
      <p>Â© 2016 Michael Kohler - JavaScript frontend and backend developer
        with a passion for Web Standards</p>
    </footer>
  );
};

export default Footer;
