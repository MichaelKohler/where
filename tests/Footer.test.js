import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../src/Footer';

test('should correctly render footer', () => {
  const component = shallow(<Footer />);
  expect(component).toMatchSnapshot();
});
