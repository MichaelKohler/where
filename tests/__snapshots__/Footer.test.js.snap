// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render footer 1`] = `
<footer>
  <p>
    Michael Kohler - Data accurate from 2013 onwards
  </p>
</footer>
`;
